easybuild/main.py

```diff
@@ -208,7 +233,8 @@ def main(args=None, logfile=None, do_build=None, testing=False):
 
     # initialise logging for main
     global _log
-    _log, logfile = init_logging(logfile, logtostdout=options.logtostdout, silent=testing or options.terse)
+    _log, logfile = init_logging(logfile, logtostdout=options.logtostdout,
+                                 silent=(testing or options.terse)), color=options.color)
```

Comment:
the `))` here should be `)`, syntax error

@riccardomurri you didn't test this, did you? Did you?! ;-)